packages/web-components/public/shell.css

```diff
@@ -0,0 +1 @@
+../../../apps/public-docsite-v9/public/shell.css
```

Comment:
please inline these changes to your particular domain. its out of scope for this gigantic PR to make cross domain implementation generic. that can be tackled later

Reply:
@miroslavstastny to add context.

Reply:
Agree with @Hotell. I suggest we exclude all changes to `apps/public-docsite-v9` (and also `.storybook/docs-root.css`) from this PR and fix WC3 storybook.